eg/gps-hardware-serial.js

```diff
@@ -0,0 +1,18 @@
+var five = require("../lib/johnny-five.js");
+var board = new five.Board();
+
+board.on("ready", function() {
+
+  /*
+   * Explicitly setting HW_SERIAL1 for the port
+   */
+  var gps = new five.GPS({
+    port: "HW_SERIAL1"
```

Comment:
This won't work because `port` is expecting the value of `SERIAL_PORT_IDs.HW_SERIAL1`, not the key itself. I'd like to make this possible, but would prefer to wait until @nebrius has settled his implementation of the serial extensions in Raspi-IO (to see how the patterns will intersect). 

For now let's drop `gps-hardware-serial.js` from `tpl/programs.json`. I don't want to discard it, but I don't want to mislead users